types/index.d.ts

```diff
@@ -14,7 +14,8 @@ type FastifyLeveldb = FastifyPluginCallback<fastifyLeveldb.FastifyLeveldbOptions
 declare namespace fastifyLeveldb {
   export interface FastifyLeveldbOptions {
     name: string,
-    path: string
+    path: string,
+    options: any
```

Comment:
Would `Record<string, unknown>` be a better type here?

Reply:
Probably, but the LevelUp library defines it as `any`, and I'm not familiar enough with it to know options are always dictionaries.